meta/requirements.yml

```diff
@@ -0,0 +1,2 @@
+collections:
+  - containers.podman
```

Comment:
Is it worth dropping a comment that the collection will be vendored downstream in RHEL?